frontend/src/pages/Users/Users.tsx

```diff
@@ -82,6 +83,14 @@ export const Users: React.FC = () => {
       id: 'acceptedTermsVersion',
       disableFilters: true
     },
+    {
+      Header: 'Last Logged In',
+      accessor: ({ lastLoggedIn }) => (lastLoggedIn ? `${formatDistanceToNow(new Date(lastLoggedIn))} ago` : "None"),
```

Comment:
Does this work? I think this might be simpler:

```suggestion
      accessor: ({ lastLoggedIn }) => (lastLoggedIn ? `${formatDistanceToNow(lastLoggedIn)} ago` : "None"),
```